contract_recurring_analytic_distribution/models/account_analytic_invoice_line.py

```diff
@@ -3,28 +3,14 @@
 # Copyright 2017 Tecnativa - Vicent Cubells
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
-from openerp import models, fields, api
```

Comment:
Just rename the file to `account_analytic_account` instead of moving part of it, as the main object is that one.